cmd/kubebuilder/create/resource/run.go

```diff
@@ -61,6 +62,7 @@ func AddCreateResource(cmd *cobra.Command) {
 	createResourceCmd.Flags().BoolVar(&nonNamespacedKind, "non-namespaced", false, "if set, the API kind will be non namespaced")
 	createResourceCmd.Flags().BoolVar(&controller, "controller", true, "if true, generate the controller code for the resource")
 	createResourceCmd.Flags().BoolVar(&generate, "generate", true, "generate source code")
+	createResourceCmd.Flags().BoolVar(&controllerOnly, "controller-only", false, "skip resources, generate controller only")
```

Comment:
hmm.... add a resource with --controller-only looks a bit confusing. "Add controller" should address this use case right ?

Reply:
I didn't realize that it might cause confusion. Do you suggest to add a new command like `add controller` or `create controller`? This new command will have almost the same flags and need to call the same functions as current `create resource`. I'm not sure if it worth to do it.

Reply:
yeah I think `create controller` makes sense.  We don't actually want to create a resource in this case.